spec/features/granules/granule_filters_spec.rb

```diff
@@ -108,6 +108,31 @@
         click_button "granule-filters-submit"
       end
 
+      it "entering and removing values doesn't set invalid query param in the url" do
```

Comment:
The fix is to prevent ```&pg[1][cc][min]=&``` from showing up in the url (a query param without any values). You can replicate the issue in prod by entering a cloud cover min value, say 30, click somewhere else to un-focus the input textbox, and clear the "30" in that input and un-focus again to trigger the query update.

I was trying to do the same thing in the test. I couldn't figure out why it didn't work.